pkg/kubectl/proxy_server.go

```diff
@@ -36,6 +37,14 @@ const (
 	DefaultMethodRejectRE = "POST,PUT,PATCH"
 )
 
+var (
+	// The reverse proxy will periodically flush the io writer at this frequency.
+	// Only matters for long poll connections like the one used to watch. With an
+	// interval of 0 the reverse proxy will buffer content sent on any connection
+	// with transfer-encoding=chunked.
```

Comment:
Add a TODO: about looking into the proxy to see if we can fix this without always adding 100ms of latency.